From 1da993745336119e747bddd2ba448dd94f77dd94 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Mon, 19 Aug 2019 21:12:21 +0200
Subject: [PATCH 063/152] workspace: Use operator shorthand

Shorthands like a += b are well-established, so prefer them over the
less concise a = a + b.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/710
---
 js/ui/workspace.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/workspace.js b/js/ui/workspace.js
index d16a5c1ca..b38a9428e 100644
--- a/js/ui/workspace.js
+++ b/js/ui/workspace.js
@@ -948,7 +948,7 @@ var LayoutStrategy = class {
             y += row.height * row.additionalScale + this._rowSpacing;
         }
 
-        compensation = compensation / 2;
+        compensation /= 2;
 
         for (let i = 0; i < rows.length; i++) {
             let row = rows[i];
-- 
2.23.0

